### ISSUE TITLE
wasm file fetching failed in electron

### ISSUE THREAD

**Issue by boocaii** [Sec-ant/zxing-wasm#80]
source code:
```
const {
  setZXingModuleOverrides,
  readBarcodesFromImageFile,
} = require("zxing-wasm");

const qr = async () => {
  const imageFile = await fetch(
    "https://api.qrserver.com/v1/create-qr-code/?size=150x150&data=Hello%20world!"
  ).then((resp) => resp.blob());
  readBarcodesFromImageFile(imageFile).then(console.log);
};

qr()
```

`npm start` outptu:
```
% npm start

> zxing-wasm@1.0.0 start
> electron .

failed to asynchronously prepare wasm: both async and sync fetching of the wasm failed
Aborted(both async and sync fetching of the wasm failed)
(node:70344) UnhandledPromiseRejectionWarning: RuntimeError: Aborted(both async and sync fetching of the wasm failed). Build with -sASSERTIONS for more info.
    at hr (/Users/jpz/GitHub/jslab/zxing-wasm/node_modules/zxing-wasm/dist/cjs/full/index.js:1:2278)
    at /Users/jpz/GitHub/jslab/zxing-wasm/node_modules/zxing-wasm/dist/cjs/full/index.js:1:2935
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
(Use `Electron --trace-warnings ...` to show where the warning was created)
(node:70344) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
```

but it's working when I run `node main.js`.

Environment:
node v20.10.0
electron v29.1.0
zxing-wasm v1.2.4

**Comment by Sec-ant** [comment]
Hi @boocaii, thanks for reporting this problem.

I can reproduce this error. But there's something I don't understand.

From the debug message I added, it seems `WebAssembly.instantiateStreaming` is not supported in electron `v29.1.0`:

![image](https://github.com/Sec-ant/zxing-wasm/assets/10386119/e802bf8c-d717-4d25-a2c9-e9c57ec6221b)

![image](https://github.com/Sec-ant/zxing-wasm/assets/10386119/be790fe7-bf56-4bf2-bf83-23f81ff75e62)

But from [this release note](https://releases.electronjs.org/release/v24.0.0-alpha.1#:~:text=instantiateStreaming), it appears to have been already supported starting from `v24.0.0-alpha.1`.

To be honest I'm not very familiar with electron so I wonder if there is something I don't know about, like some flags have to be set or a different mode has to be configured, etc.

If this doesn't ring a bell to you either, I'll try to apply a patch like [what I did to support Bun](https://github.com/Sec-ant/zxing-wasm/commit/8b39051653955e25a1b27dba64135a20b4cdc4c0) to fix this issue.
